clients/ts/FunctionalTests/ts/Utils.ts

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
```

Comment:
Removed because....?

Reply:
We don't need it anymore. The hangs were caused by that bug I fixed not the default 5 second timeout.